homeassistant/components/proxmoxve/__init__.py

```diff
@@ -143,43 +103,93 @@ def build_client() -> ProxmoxAPI:
         host_name = host_config["host"]
         coordinators[host_name] = {}
 
-        proxmox_client = hass.data[PROXMOX_CLIENTS][host_name]
+    # import to config flow
+    if DOMAIN in config:
+        for conf in config[DOMAIN]:
+            hass.async_create_task(
+                hass.config_entries.flow.async_init(
+                    DOMAIN, context={"source": SOURCE_IMPORT}, data=conf
+                )
+            )
```

Comment:
I'm kinda confused? Why was the integration set up above and imported here? Then it sets up again after import right?

`async_setup` should only handle import at this point.